### ISSUE TITLE
Train ADEChallengeDataset,But train loss is small,But validation loss is too big.

### ISSUE THREAD

**Issue by lily10086** [DeepSegment/ConvNet-Segment#7]
I train ADEChallengeDataet many times,But validation loss is too big,Is it normal?

2017-11-17 17:42:07.124816 ---> Validation_loss: 1.42196
Step: 8310, Train_loss:1.06558
Step: 8320, Train_loss:1.16616
Step: 8330, Train_loss:1.4604
Step: 8340, Train_loss:1.15065
Step: 8350, Train_loss:1.36698
Step: 8360, Train_loss:1.07944
Step: 8370, Train_loss:1.29012
Step: 8380, Train_loss:1.33168
Step: 44400, Train_loss:0.493773
2017-11-18 22:20:16.527388 ---> Validation_loss: 1.52609
Step: 44410, Train_loss:0.43549
Step: 44420, Train_loss:0.456891
Step: 44430, Train_loss:0.499639
Step: 44440, Train_loss:0.547243
Step: 44450, Train_loss:0.615641
Step: 44460, Train_loss:0.433227
Step: 44470, Train_loss:0.437496
Step: 44480, Train_loss:0.526902
Step: 44490, Train_loss:0.47248
Step: 44500, Train_loss:0.48162
2017-11-18 22:24:56.463316 ---> Validation_loss: 2.35229
......
....

**Comment by txzhao** [comment]
It seems normal to me but maybe some overfitting problems may exist due to unsatisfactory parameter settings. Please try a different one (especially the learning rate) and see what's different. In fact, I also met with similar results. One technique I suggest is doing gird search or random search for some hyper-parameters but that's gonna take a long time.

**Comment by lily10086** [comment]
Oh,Thank you very much!!,You always give me much help!

**Comment by txzhao** [comment]
no problem
